tests/utils/test_play.py

```diff
@@ -22,6 +22,9 @@ class DummyEnvSpec:
 
 
 class DummyPlayEnv(gym.Env):
+    def __init__(self, render_mode=None):
```

Comment:
Type annotation for the argument? I know it's only a test, but let's keep it clean. I think it'd be `Optional[str]`, right?

Reply:
yes, you are right, thanks